model/test/user.json

```diff
@@ -0,0 +1,10 @@
+[
+  {
+    "type": "go",
+    "output": "."
+  },
+  {
+    "type": "ts",
+    "output": "${PROJECT}/model"
```

Comment:
where is PROJECT set?  in the github action?

Reply:
Originally thought I'd use a shell script to set the environment variable and run scaffold. I haven't needed that because relative paths have worked.